tests/accumulator_test.py

```diff
@@ -263,17 +264,41 @@ class SumAccumulatorTest(unittest.TestCase):
 
     def test_without_noise(self):
         sum_accumulator = accumulator.SumAccumulator(
-            accumulator.SumParams(), list(range(6)))
+            accumulator.SumParams(
+                MeanVarParams(1, 1, 0, 0, 1, 1,
+                              pipeline_dp.NoiseKind.GAUSSIAN)), list(range(6)))
         self.assertEqual(sum_accumulator.compute_metrics(), 15)
 
         sum_accumulator.add_value(5)
         self.assertEqual(sum_accumulator.compute_metrics(), 20)
 
         sum_accumulator_2 = accumulator.SumAccumulator(
-            accumulator.SumParams(), list(range(3)))
+            accumulator.SumParams(
+                MeanVarParams(1, 1, 0, 0, 1, 1,
+                              pipeline_dp.NoiseKind.GAUSSIAN)), list(range(3)))
         sum_accumulator.add_accumulator(sum_accumulator_2)
         self.assertEqual(sum_accumulator.compute_metrics(), 23)
 
+    def test_with_noise(self):
+        sum_accumulator = accumulator.SumAccumulator(
+            accumulator.SumParams(
+                MeanVarParams(2, 1, 1, 1, 1, 3,
```

Comment:
you can set higher eps (like 100), then you can use smaller, delta in `assertAlmostEqual`

Reply:
Done, thanks for the tip